### ISSUE TITLE
A11y_SQLAzureDataStudioMAC_Extensions - install_Keyboard: The tooltip info is not available for the button 'Manage' using keyboard arrow keys unlike mouse.

### ISSUE THREAD

**Issue by bbarik06** [microsoft/azuredatastudio#19876]
**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev3;#BM_SQLAzureDataStudioMAC_June2022;#SQL Azure Data Studio;#MAC;#FTP;#DesktopApp;#Keyboard;#AINotInScope;#WCAG1.1.1;

## Environment Details:
Application Name: SQL Azure Data Studio
Application Version: Version 1.37.0
Mac Version: macOS Monterey 12.4

## Repro Steps:

1. Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility tester after verification of the fix.
2. Launch SQL Azure data studio.
3. Connect to a server.
4. Tab to 'Extensions' button and select.
5. Tab till 'Manage' button beside 'Uninstall'.
6. Use keyboard to observe the tooltip info since visual label is not available.

## Actual:
The tooltip info is not available for the button 'Manage' using keyboard arrow keys unlike mouse.

## Expected:
The tooltip info should be available for the button 'Manage' using keyboard arrow keys like mouse since there are no visual label present and on using keyboard esc key to dismiss the tooltip the focus should stay on the 'Manage' button.

## User Impact:
Keyboard users will face difficulty in accessing the important tooltip information if not available and might not be able to recognize a particular control especially in absence of visual label.

## Attachment:
![17_Tooltip not available](https://user-images.githubusercontent.com/98393966/176432767-a5739828-9d2c-4233-86e6-ea56f35c1d8b.gif)

**Comment by alanrenmsft** [comment]
similar to this one: https://github.com/microsoft/azuredatastudio/issues/19566
please downgrade to sev3
